<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HedgehogEdit</class>
 <widget class="QFrame" name="HedgehogEdit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="leName"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="toolButton">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="cbLevel">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <item>
      <property name="text">
       <string>Human</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Level 1</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="leHat"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Level</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="sbHealth">
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>300</number>
     </property>
     <property name="value">
      <number>100</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Health</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Hat</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="pbCoordinates">
     <property name="text">
      <string>Random</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Place</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
